modules/akka/src/main/scala/ch/epfl/bluebrain/nexus/rdf/akka/UriOps.scala

```diff
@@ -0,0 +1,19 @@
+package ch.epfl.bluebrain.nexus.rdf.akka
+
+import akka.http.scaladsl.model.Uri
+import ch.epfl.bluebrain.nexus.rdf.Iri
+
+trait UriOps {
+
+  /**
+    * Implicit conversion for constructing an [[Iri]] from the provided [[Uri]]
+    */
+  final implicit def uriToIri(uri: Uri): Iri = Iri.unsafe(uri.toString())
+
+  /**
+    * Syntactic sugar for constructing an [[Iri]] from the [[Uri]]
+    */
+  implicit final class UriSyntax(uri: Uri) {
```

Comment:
`implicit final class UriSyntax(private val uri: Uri) extends AnyVal {...}`

Reply:
let's use the same encoding as the ones used in cats: https://github.com/typelevel/cats/blob/master/core/src/main/scala/cats/syntax/applicative.scala